### ISSUE TITLE
Error after login to Hub: Syntax error or access violation

### ISSUE THREAD

**Issue by poppabear8883** [getcandy/getcandy#9]
```
Illuminate\Database\QueryException

SQLSTATE[42000]: Syntax error or access violation: 1055 'karve.getcandy_order_addresses.contact_email' isn't in GROUP BY
 (SQL: select COUNT(*) count, `getcandy_order_addresses`.`contact_email` from `getcandy_order_addresses` left join 
getcandy_order_addresses address_join on `address_join`.`contact_email` = `getcandy_order_addresses`.`contact_email` where 
`getcandy_order_addresses`.`type` = billing and `getcandy_order_addresses`.`created_at` between 2021-12-20 00:00:00 and 
2021-12-26 00:00:00 and `getcandy_order_addresses`.`contact_email` is not null group by `getcandy_order_addresses`.`id`) 
(View: /home/vagrant/projects/karve/vendor/getcandy/getcandy/resources/views/livewire/dashboard.blade.php)
```

**Comment by poppabear8883** [comment]
Possibly a `mariadb` vs `mysql` compatibility issue ?

**Comment by glennjacobs** [comment]
Looks like an easy fix. Will sort in the morning.

**Comment by alecritson** [comment]
This should be resolved in [2.0-beta4](https://github.com/getcandy/getcandy/releases/tag/2.0-beta4) If it's still an issue please re open :)
